atuin-client/config.toml

```diff
@@ -22,20 +24,16 @@
 ## for example: "+9", "-05", "+03:30", "-01:23:45", etc.
 # timezone = "local"
 
-## enable or disable automatic sync
-# auto_sync = true
-
 ## enable or disable automatic update checks
 # update_check = true
 
-## address of the sync server
-# sync_address = "https://api.atuin.sh"
+## Timeout (in seconds) for acquiring a local database connection (sqlite)
+# local_timeout = 5
```

Comment:
tbh I'd like to eliminate all top-level/root config items

Reply:
I also think that all config settings should belong to a `[section]`. If one is not sure, it's still possible to create a `Misc` or `Other` section. On the other hand, if it is not clear, maybe the config option is not useful in the first place (with exceptions of course).